src/components/Todo/Todo.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './Todo.css';
+import PropTypes from 'prop-types';
+
+import { User } from '../User';
+
+export const Todo = ({ todo }) => (
+  <>
+    <div className="border">
+      <input type="checkbox" defaultChecked={todo.completed} />
```

Comment:
```suggestion
      <input type="checkbox" сhecked={todo.completed} />
```
і тоді onChange міняти в state статус todo